ext/liquid_c/raw.c

```diff
@@ -30,10 +30,18 @@ static bool match_full_token_possibly_invalid(token_t *token, struct full_token_
     for (long i = len - 3; i >= 0; i--) {
         char c = str[i];
 
-        if (is_word_char(c)) {
+        // match \s
+        bool is_whitespace = (
+            c == ' ' || c == '\t' || c == '\r' || c == '\n' || c == '\v' || c == '\f'
+        );
```

Comment:
We can use the `ISSPACE` macro (which wraps the C function [`isspace`](https://devdocs.io/c/string/byte/isspace)) instead.